.github/workflows/ci.yml

```diff
@@ -21,4 +21,4 @@ jobs:
         run: npm install
 
       - name: Run tests
-        run: npm test
+        run: xvfb-run -a npm test
```

Comment:
xvfb is only needed for Linux. If this gets updated to run on Windows and MacOS, we can change to the following to delineate between platforms:
```
   - run: xvfb-run -a npm test
      if: runner.os == 'Linux'
    - run: npm test
      if: runner.os != 'Linux'
```